Sketch.java

```diff
@@ -6,31 +6,116 @@ public class Sketch extends PApplet {
   /**
    * Called once at the beginning of execution, put your size all in this method
    */
+
+
+    //clock variables
+    float clockX, clockY;
+    float secondsCircle;
+    float minutesCircle;
+    float hoursCircle;
+    float clock;
+
+
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(1000, 1000);
   }
 
-  /** 
-   * Called once at the beginning of execution.  Add initial set up
-   * values here i.e background, stroke, fill etc.
-   */
-  public void setup() {
-    background(210, 255, 173);
+  
+   public void setup() {
```

Comment:
drawing should not occur in the setup() method. Initializing variables should occur there, not drawing